agent/handlers/agentapi/taskprotection/v1/types/types.go

```diff
@@ -21,14 +21,14 @@ import (
 // Protection for a Task
 type taskProtection struct {
 	protectionEnabled bool
-	expiresInMinutes  *int
+	expiresInMinutes  *int64
```

Comment:
why do we need this to be an int64?  (max int64 minutes converted to years would end up being 17548272520652.16 years...)

Reply:
This is to match the control plane type: https://github.com/aws/amazon-ecs-agent/blob/feature/task_scale_in_protection_pass_through/agent/ecs_client/model/ecs/api.go#L14110